dask/array/tests/test_fft.py

```diff
@@ -27,9 +31,21 @@ def key(k):
     "ihfft",
 ]
 
+all_nd_funcnames = [
+    "fft2",
+    "ifft2",
+    "fftn",
+    "ifftn",
+    "rfft2",
+    "irfft2",
+    "rfftn",
+    "irfftn",
+]
+
 nparr = np.arange(100).reshape(10, 10)
 darr = da.from_array(nparr, chunks=(1, 10))
 darr2 = da.from_array(nparr, chunks=(10, 1))
+darr3 = da.from_array(nparr, chunks=(10, 10))
```

Comment:
That's a nice test. How long did it take to run?

We should probably also test at least one case of a repeated axis. IMHO I wouldn't want to combine this with the wrapper test and just test the exposed NumPy interface.

Side note: We don't currently support `norm` in the wrapper. So we should probably just drop that from the test.